### ISSUE TITLE
Theme Scaffold command did not generate views/default.blade.php

### ISSUE THREAD

**Issue by azulkipli** [AsgardCms/Platform#151]
I got this error after activated new generated theme
```php
InvalidArgumentException in compiled.php line 13828:
View [default] not found.
in compiled.php line 13828
at FileViewFinder->findInPaths('default', array('/Users/azul/WebRoot/laravel/asgard/resources/views', '/Users/azul/WebRoot/laravel/asgard/Themes/Librand/views/')) in compiled.php line 13801
at FileViewFinder->find('default') in compiled.php line 13929
```
I checked that theme scaffold command did not generate "themename/views/default.blade.php"

**Comment by nWidart** [comment]
indeed, it creates an index.blade.php, I'll rename that now to default. :)

**Comment by nWidart** [comment]
Fixed on [1.13.1](https://github.com/AsgardCms/Workshop/releases/tag/1.13.1)

Thanks for reporting the issue. :smile:
